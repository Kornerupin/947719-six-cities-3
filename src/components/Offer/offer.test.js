import Offer from './offer';
import offersMock from '../../mocks/offers-mock';

it(`<Offer /> Should render one offer`, () => {
  const tree = renderer.create(
      <Offer
        offer={offersMock[0]}
        onOfferMouseOver={() => { }}
        onOfferNameClick={() => { }}
      />).toJSON();

  expect(tree).toMatchSnapshot();
});
