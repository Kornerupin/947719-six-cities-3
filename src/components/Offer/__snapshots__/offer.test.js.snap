// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Offer /> Should render one offer 1`] = `
<article
  className="cities__place-card place-card"
  onMouseOver={[Function]}
>
  <div
    className="cities__image-wrapper place-card__image-wrapper"
  >
    <a
      href="#"
    >
      <img
        alt="Place image"
        className="place-card__image"
        height="200"
        src=""
        width="260"
      />
    </a>
  </div>
  <div
    className="place-card__info"
  >
    <div
      className="place-card__price-wrapper"
    >
      <div
        className="place-card__price"
      >
        <b
          className="place-card__price-value"
        >
          €
        </b>
        <span
          className="place-card__price-text"
        >
          / night
        </span>
      </div>
      <button
        className="place-card__bookmark-button place-card__bookmark-button--active button"
        type="button"
      >
        <svg
          className="place-card__bookmark-icon"
          height="19"
          id="icon-bookmark"
          viewBox="0 0 17 18"
          width="18"
        >
          <path
            d="M3.993 2.185l.017-.092V2c0-.554.449-1 .99-1h10c.522 0 .957.41.997.923l-2.736 14.59-4.814-2.407-.39-.195-.408.153L1.31 16.44 3.993 2.185z"
          />
        </svg>
        <span
          className="visually-hidden"
        >
          In bookmarks
        </span>
      </button>
    </div>
    <div
      className="place-card__rating rating"
    >
      <div
        className="place-card__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "NaN%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <h2
      className="place-card__name"
      onClick={[Function]}
    >
      <a
        href="#"
      />
    </h2>
    <p
      className="place-card__type"
    />
  </div>
</article>
`;
